core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -340,4 +347,24 @@ private[mesos] trait MesosSchedulerUtils extends Logging {
     sc.conf.getTimeAsSeconds("spark.mesos.rejectOfferDurationForUnmetConstraints", "120s")
   }
 
+  protected def getExecutorRelatedPort(sc: SparkContext, configProperty: String): Int = {
+    // either: spark.blockManager.port or spark.executor.port
+    sc.conf.getInt(s"$configProperty", 0)
+  }
+
+  protected def checkPorts(sc: SparkContext, ports: List[(Long, Long)]): Boolean = {
```

Comment:
Judging from pre-existing methods not all of them have a proper javadoc with parameters etc so it seemed to me a bit ad hoc the whole documenting process. I am confused.
Compare def startServiceOnPort doc versus any other method in Utils. Also i dont see much in the scala style doc. Maybe i am wrong...